pandas/io/tests/test_ga.py

```diff
@@ -142,7 +145,8 @@ def test_segment(self):
                 end_date=end_date,
                 segment=-2,
                 dimensions=['date', 'hour'],
-                parse_dates={'ts': ['date', 'hour']})
+                parse_dates={'ts': ['date', 'hour']},
+                index_col=0)
 
             assert isinstance(df, DataFrame)
```

Comment:
can you fix these tests to use either:

``self.assertIsInstance(....)`` testing or

just compare vs an expected frame (e.g. ``assert_frame_equal``)